### ISSUE TITLE
Pie chart slices order is reversed

### ISSUE THREAD

**Issue by windkomo** [chartist-js/chartist#700]
Hi,

I noticed that the order of slices is reversed when compared to the labels,.

For instance, see the SIMPLE PIE CHART example from https://gionkunz.github.io/chartist-js/examples.html.

Array:

``` javascript
var data = {
    series: [5, 3, 4]
};
```

Order is 5, 3, 4

Slices:

``` html
<g class="ct-series ct-series-c"><path d="[...]" class="ct-slice-pie" value="4"></path></g>
<g class="ct-series ct-series-c"><path d="[...]" class="ct-slice-pie" value="3"></path></g>
<g class="ct-series ct-series-c"><path d="[...]" class="ct-slice-pie" value="5"></path></g>
```

Order is 4, 3, 5

Labels:

``` html
<text dx="[...]" dy="[...]" text-anchor="middle" class="ct-label">42%</text>
<text dx="[...]" dy="[...]" text-anchor="middle" class="ct-label">25%</text>
<text dx="[...]" dy="[...]" text-anchor="middle" class="ct-label">33%</text>
```

Order is 5, 3, 4

Is this intentional ?
I need that the label of a slice to be also hovered when the slice is hovered so did something like: 

``` javascript
// On chart draw
$slices.on('mouseover', function() {
    var index = $slices.index($(this));
    $container.find('.ct-label').eq(index).addClass('hovered');
});
```

But then realized that the slices and labels indexes were not matching. I think I can just use `reverse` or something but was curious about this.

**Comment by gionkunz** [comment]
I guess it would be better if the order would be the same. Let's keep this issue as a bug.

**Comment by FlavienBusseuil** [comment]
Hi,

This is the reason the order was changed: https://github.com/gionkunz/chartist-js/issues/31

This fix re-opened the initial question in #31 (Rounded Corner for Donut Chart). Any new suggestion to solve this particular design?

Thanks.

**Comment by FlavienBusseuil** [comment]
In the meanwhile, the following code is working for me:
```
donut.on("created", () => {
	const svg = donut.container.firstChild;
	svg.insertBefore(svg.lastChild, svg.firstChild);
});
```
